CMakeModules/before_project_setup.cmake

```diff
@@ -0,0 +1,20 @@
+# ===========================================================================
+# CMAKE PART: BUILD-OPTIONS: Setup CMAKE Build Config
+# ===========================================================================
+# USE: INCLUDE-BEFOR project() !!!
+
+include_guard(DIRECTORY)
+
+# ---------------------------------------------------------------------------
+# use ccache if found
+# ---------------------------------------------------------------------------
+find_program(CCACHE_EXECUTABLE "ccache" HINTS /usr/local/bin /opt/local/bin)
```

Comment:
/usr/local is automatically in the search list on Unix, and /opt/local on Darwin, so the HINTS should not be necessary.

Reply:
thanks, I change it